cypress/support/pages/administration/pageAdministration.js

```diff
@@ -18,6 +18,10 @@ class Administration {
     static #cancelButtonConfirmation = '.cancel-modal button.btn-close'
     static #deleteButtonCancel = '.cancel-modal button.historyback'
     static #deleteButtonConfirmation = '.delete-modal.in  button.btn-submit'
+    static #newSchoolAdminPageButton = '.btn-info'
+    static #chatToggleSwitch = '.rocketchat-switch'
+    static #videoconferenceToggleSwitch = '.videoconference-switch'
+    static #saveGeneralSettingsButton = '.my-5'
     static #tableContents =  '[data-testid="table-data-body"]'
 
     clickOnFAB () {
```

Comment:
method clickOnFAB is here explicit for deleteButtonConfirmation, but in Administration there also could be other functions on FAB buttons, I think. Shouldn't we use a specific name for this method (clickOnDeletionFAB) or us a parameter in it?

Reply:
Actually it's not for deletion but rather to get to the mask where a user can be added to the application.
I understand your concern and agree on it. Thing is, it's always a fab (floating action button) at first, which then has different possibilities within it.
I could rename it to clickOnFABForUsers?!?

Reply:
Mein Fehler ;-)